src/commands/generate/serviceGenerator.ts

```diff
@@ -137,6 +137,21 @@ export function generateService(
             scope: Scope.Public,
             docs: docs != null ? [docs] : undefined,
         });
+
+        endpointDefinition.errors?.forEach(error => {
+            imports.push(
```

Comment:
Is it possible that we get multiple imports for the same error in the same file?

Reply:
No, imports are deduped before that are added. I also added a test to make sure that we don't do duplicate imports.